src/actions/commentActions.js

```diff
@@ -0,0 +1,29 @@
+import axios from '../helpers/axiosInstance';
+
+export const fetchCommentsAction = slug => (dispatch) => {
+  axios
+    .get(`${process.env.baseURL}/articles/${slug}/comments`)
+    .then((response) => {
+      dispatch({
+        type: 'FETCH_COMMENTS',
+        payload: response.data.comments,
+      });
+    });
+};
+
+export const replyToCommentAction = (slug, commentId, commentReply) => (dispatch) => {
+  const data = {
+    comment: {
+      body: commentReply,
+    },
+  };
+
+  axios
+    .post(`${process.env.baseURL}/articles/${slug}/comments/${commentId}/reply`, data)
+    .then((response) => {
+      location.reload();
+    })
+    .catch((error) => {
+      console.log(error);
```

Comment:
@ezrogha  remove the console.log`
Because we agreed that, we should not be leaving them anywhere in our code

Reply:
Thank you for that feedback,
@felixkiryowa